specification/connectedvmware/resource-manager/readme.csharp.md

```diff
@@ -11,5 +11,5 @@ csharp:
   clear-output-folder: true
   client-side-validation: false
   namespace: Microsoft.ConnectedVMwarevSphere
-  output-folder: $(csharp-sdks-folder)/ConnectedVMware/Microsoft.Azure.Management.ConnectedVMwarevSphere/src/Generated
+  output-folder: $(csharp-sdks-folder)/connectedvmware/Microsoft.ConnectedVMwarevSphere/src/Generated
```

Comment:
May I know why change to "Microsoft.ConnectedVMwarevSphere"?
As I know the namespace of other RPs is in shape of "Microsoft.Azure.Management.xxxx". Is it possible to use this form?

Reply:
I had talked to one the member of NET sdk team, he has suggested to go with this name, as they have check1 and check2 in check1 sdk they follow this convention. You can see here in compute RP they have both: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute  

Please suggest me what finally should i keep?

Reply:
@fengzhou-msft  could you please review this change.

Reply:
Please change namespace to `Microsoft.Azure.Management.ConnectedVMwarevSphere`. For `output-folder` only change to lower case for `connectedvmware`.

Reply:
Changed namespace name.